Apollo.Dockerfile

```
@@ -0,0 +1,17 @@
+FROM node:20-alpine3.18
+
+RUN npm install -g pnpm
```

Hadolint (2.12.0)

[warning] Pin versions in npm. Instead of `npm install <package>` use `npm install <package>@<version>`

(DL3016)